Plugin.pm

```diff
@@ -521,20 +521,27 @@ sub QobuzArtist {
 			passthrough => [{
 				artistId  => $artist->{id},
 			}],
-		},{
-			name => cstring($client, 'PLUGIN_QOBUZ_ADD_FAVORITE', $artist->{name}),
-			image => 'html/images/favorites.png',
-			url  => \&QobuzAddFavorite,
-			passthrough => [{
-				artist_ids => $artist->{id},
-			}],
-			nextWindow => 'parent'
 		};
-;
-
-		$cb->( {
-			items => $items
-		} );
+		
+		Plugins::Qobuz::API->getUserFavorites(sub {
+			my $favorites = shift;
+			my $artistId = $artist->{id};
+			my $isFavorite = grep { $_->{id} eq $artistId } @{$favorites->{artists}->{items}};
```

Comment:
Please make sure `$favorites->{artists}->{items}` does exist and is a list before accessing it.

Reply:
See latest commit.